case-lib/lib.sh

```diff
@@ -47,11 +47,20 @@ func_lib_setup_kernel_last_line()
 SOF_LOG_COLLECT=0
 func_lib_start_log_collect()
 {
-    local is_etrace=${1:-0} ldcFile
-    ldcFile=/etc/sof/sof-$(sof-dump-status.py -p).ldc || {
-        >&2 dlogw "sof-dump-status.py -p to query platform failed"
-        return
+    local is_etrace=${1:-0}
+    # if user doesn't specify ldcFile, fall back to /etc/sof/sof-PLATFORM.ldc,
+    # which is used by CI.
+    [[ -n "$ldcFile" ]] || {
+        ldcFile=/etc/sof/sof-$(sof-dump-status.py -p).ldc || {
+            >&2 dlogw "Failed to query platform with sof-dump-status.py"
+            return 1
+        }
+    }
+    [[ -e "$ldcFile" ]] || {
+        dlogw "LDC file not found, check ldcFile environment variable or put your sof-*.ldc to /etc/sof"
```

Comment:
`$ldcFile` is the main information, it must be in the error message.

Reply:
fixed.  I agree with your point, the log should be readable rather than  beautiful, but I also think the original author may make a lot of compromise to redirect the sof-logger's stderr. The purpose of this PR is making the log more readable.

Reply:
I still don't see `$ldcFile` in the message.